content/docs/concepts/security/index.md

```diff
@@ -461,10 +460,24 @@ peers:
   - mtls: {}
 {{< /text >}}
 
-Currently, the mutual TLS setting doesn't require any parameters. Hence,
-`-mtls: {}`, `- mtls:` or `- mtls: null` declarations are treated the same. In
-the future, the mutual TLS setting may carry arguments to provide different
-mutual TLS implementations.
+The mutual TLS setting has an optional `mode` parameter that defines the
+strictness of the peer transport authentication:
+
+- `mode: STRICT` enforces that all peers must use mutual TLS. This declaration
+is identical to `-mtls: {}`, `- mtls:` or `- mtls: null`
```

Comment:
It would be better if you link to the reference doc that has all the declarations instead but this is not a deal breaker.

Reply:
I've linked to the reference docs, but I think it's key to stress that `STRICT` is default mode and that if you don't specify a mode it's equivalent to specifying `STRICT` mode